TestAssets/TestProjects/FilterProfiles/NewtonsoftMultipleVersions.xml

```diff
@@ -0,0 +1,4 @@
+<CacheArtifacts>
+  <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  <PackageReference Include="Newtonsoft.Json" Version ="9.0.2-beta2"/>
```

Comment:
I don't like `Include` here since that is an msbuild concept, I think we should make it even more obvious that this is its own format, separate from csproj package references.

This is my suggestion:
``` XML
<CacheArtifacts>
  <Package Id="NewtonSoft.Json" Version="9.0.1" />
</CacheArtifacts>
```